properties/service_fabric_managed_prod.props

```diff
@@ -4,6 +4,10 @@
   
   <Import Project="$(MSBuildThisFileDirectory)service_fabric_managed_common.props" />
 
+  <PropertyGroup>
```

Comment:
You've added this to service_fabirc_managed.common.props so please remove form here as its not overriding with new value.